terraform/production/task_settings.json

```diff
@@ -0,0 +1,112 @@
+{
+    "TargetMetadata": {
+        "TargetSchema": "",
+        "SupportLobs": true,
+        "FullLobMode": false,
+        "LobChunkSize": 0,
+        "LimitedSizeLobMode": true,
+        "LobMaxSize": 32,
+        "InlineLobMaxSize": 0,
+        "LoadMaxFileSize": 0,
+        "ParallelLoadThreads": 0,
+        "ParallelLoadBufferSize": 0,
+        "BatchApplyEnabled": false,
+        "TaskRecoveryTableEnabled": false,
+        "ParallelLoadQueuesPerThread": 0,
+        "ParallelApplyThreads": 0,
+        "ParallelApplyBufferSize": 0,
+        "ParallelApplyQueuesPerThread": 0
+    },
+    "FullLoadSettings": {
+        "TargetTablePrepMode": "DROP_AND_CREATE",
+        "CreatePkAfterFullLoad": false,
+        "StopTaskCachedChangesApplied": false,
+        "StopTaskCachedChangesNotApplied": false,
+        "MaxFullLoadSubTasks": 8,
+        "TransactionConsistencyTimeout": 600,
+        "CommitRate": 10000
+    },
+    "Logging": {
+        "EnableLogging": false,
```

Comment:
As this is production, should we enable logging here?

Reply:
Yep, missed that too :) Thanks!